console/network/src/helpers/id.rs

```diff
@@ -210,3 +211,17 @@ impl<F: FieldTrait, const PREFIX: u16> Distribution<AleoID<F, PREFIX>> for Stand
         AleoID::<F, PREFIX>(Uniform::rand(rng))
     }
 }
+
+impl<F: FieldTrait, const PREFIX: u16> Ord for AleoID<F, PREFIX> {
+    #[inline]
+    fn cmp(&self, other: &Self) -> Ordering {
+        self.0.cmp(&other.0)
+    }
+}
+
+impl<F: FieldTrait, const PREFIX: u16> PartialOrd for AleoID<F, PREFIX> {
+    #[inline]
+    fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
+        Some(self.0.cmp(&other.0))
```

Comment:
I'm wondering if it wouldn't be more clear to add a `PartialOrd` bound to the `Bech32ID` trait and reuse its impl here instead (i.e. `self.0.partial_cmp(&other.0)`

Reply:
Cool, is that some kind of default impl? 2454c874a

Reply:
yep, looks good :ok_hand: